src/components/header/Header.css

```diff
@@ -0,0 +1,19 @@
+.header{
+  width: 100vw;
+  height: 70px;
+  text-align: center;
+  font-size: 18px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background-color: rgb(245, 244, 243);
+}
+
+.logo{
+  margin-right: 15px;
+  height: 50px;
+}
+.logo:hover{
+  transition: 0.2s;
+  scale: 1.1;
```

Comment:
scale is not a direct property, it's the value for transform property

Use should replace it with:  transform: scale(1.1);